lib/analyzer/applications/node.ts

```diff
@@ -0,0 +1,116 @@
+import * as path from "path";
+import * as lockFileParser from "snyk-nodejs-lockfile-parser";
+
+import * as types from "../../types";
+
+const TYPE = "node";
+const VERSION = "1.0.0";
+
+export interface NodeScanResult extends types.ScanResult {
+  type: string;
+  version: string;
+  data: NodeData;
+}
+
+interface NodeData {
```

Comment:
we should consider somehow having a map somewhere in the plugin between the app type, app version and the interface itself, so maybe Registry can use it as well? instead of defining things in 2 places

Reply:
https://github.com/snyk/scan-schemas is expected to host that